models/issues/milestone.go

```diff
@@ -143,20 +170,32 @@ func GetMilestoneByRepoID(ctx context.Context, repoID, id int64) (*Milestone, er
 	} else if !has {
 		return nil, ErrMilestoneNotExist{ID: id, RepoID: repoID}
 	}
+	var labels []*Label
```

Comment:
Labels should be loaded when necessary.

Reply:
Does it means they need to be loaded in [ListMilestoneLabels](https://github.com/go-gitea/gitea/blob/eaae336808cb80cf5628f540d0f877bc171458ac/routers/api/v1/repo/milestone_label.go#L17)?

Reply:
You can have a `LoadLabels` methods to lazyly load labels.